test/authz-filler/main.go

```diff
@@ -0,0 +1,92 @@
+// A quick way to fill up a database with a large number of authz objects, in
+// order to manually test the performance of the expired-authz-purger.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/core"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type fillerConfig struct {
+	Filler struct {
+		cmd.DBConfig
+		Parallelism uint
+	}
+}
+
+type model struct {
+	core.Authorization
+
+	LockCol int
+}
+
+func main() {
+	configPath := flag.String("config", "config.json", "Path to Boulder configuration file")
+	flag.Parse()
+
+	configJSON, err := ioutil.ReadFile(*configPath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to read config file '%s': %s\n", *configPath, err)
+		os.Exit(1)
+	}
+
+	var config fillerConfig
+	err = json.Unmarshal(configJSON, &config)
+	cmd.FailOnError(err, "Failed to parse config")
+
+	// Configure DB
+	dbURL, err := config.Filler.DBConfig.URL()
+	cmd.FailOnError(err, "Couldn't load DB URL")
+	dbMap, err := sa.NewDbMap(dbURL, 1000)
```

Comment:
Can you make the `1000` a `maxOpenConns` var or comment what it is? It seems a little magic in-place and requires looking at the `sa/database.go:NewDbMap` function signature to realize its for max open conns.